apps/ledger-live-desktop/src/renderer/modals/Send/steps/Confirmation/NodeError/TechnicalErrorSection.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { Flex, Text, Icons } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+import { useCopyToClipboard } from "~/newArch/hooks/useCopyToClipboard";
+
+type Props = {
+  error: Error;
+  onSaveLogs: () => void;
+};
+
+const InteractFlex = styled(Flex)`
+  &:hover {
+    cursor: pointer;
+    background-color: ${({ theme }) => theme.colors.palette.opacityDefault.c10};
+  }
+  padding: 8px;
+  background-color: ${({ theme }) => theme.colors.palette.opacityDefault.c05};
+  border-radius: 8px;
+  column-gap: 8px;
+  align-items: center;
+
+  &:active {
+    background-color: ${({ theme }) => theme.colors.palette.opacityDefault.c20};
+  }
+`;
+
+const StyledText = styled(Text)`
+  word-break: break-all;
```

Comment:
Why the word-break ?
<img width="288" alt="image" src="https://github.com/user-attachments/assets/fde05d5c-b881-42b5-a501-52a52cdfc9c0">
That's just my opinion but I think the line break between network and error would look better.

Reply:
Could look weird with strange error message I think like long words this would go directly to next line